store/index.js

```diff
@@ -36,25 +36,25 @@ export const state = () => ({
         "Age": {
 
             componentName: "annot-continuous-values",
-            explanation: "This is an explanation for how to annotate age.",
+            explanation: "Review the annotated columns for Age. Mark missing values. Select a transformation heuristic, check the preview, and finalize the harmonization",
```

Comment:
maybe add a top level comment to remind us that strings will get split by `.` and then turned into bullet points. So I don't have to remember / look up again.